TicTacToe/Driver.fs

```diff
@@ -15,10 +15,10 @@ let keepWindowOpen () =
 let printToScreen (output) =
     printfn "%s" output
 
-let rec runGame (print, input, size, doesComputerGoFirst, characters : char list,isInverted,isMinimax) =
-    let x = startNewGame(size,doesComputerGoFirst,characters.[0], characters.[1], enterInput, printToScreen,isInverted,isMinimax)
+let rec runGame (print, input, size, doesComputerGoFirst, characters : char list,isInverted,algorithmSeed) =
```

Comment:
Adding this new parameter all over the place is a good example of a situation where 1 (seemingly) small change echoes throughout the code, potentially causing new problems. Whenever you have to do this, it's a big warning sign that things are too coupled together.